prow/config/jobs/enhancements-1.21.yaml

```
@@ -0,0 +1,313 @@
+branches:
+- release-1.21
+env:
+- name: BUILD_WITH_CONTAINER
+  value: "0"
+- name: BUILD_WITH_CONTAINER
+  value: "0"
```

Example of double BUILD_WITH_CONTAINER
```suggestion
- name: BUILD_WITH_CONTAINER
  value: "0"
```